src/utils/selectors.ts

```diff
@@ -243,3 +250,36 @@ export const noopSelector: SparkleSelector<undefined> = () => undefined;
 
 export const emptyArray = [];
 export const emptyArraySelector = <T>(): T[] => emptyArray;
+
+// AnimateMap
+export const animateMapStageOptionsSelector: SparkleSelector<AnimateMapStageOptions> = (
+  state
+) => state.animatemap.stageOptions;
+
+export const animateMapWorldBoundsSelector: SparkleSelector<AnimateMapWorldBounds> = (
+  state
+) => {
+  return {
+    width: state.animatemap.worldWidth,
+    height: state.animatemap.worldHeight,
+  };
+};
+
+export const animateMapZoomSelector: SparkleSelector<number> = (state) =>
+  state.animatemap.zoom;
+
+export const animateMapUsersSelector: SparkleSelector<
+  Map<string, ReplicatedUser>
+> = (state) => state.animatemap.users;
+
+export const animateMapVenuesSelector: SparkleSelector<
+  Map<string, ReplicatedVenue>
+> = (state) => state.animatemap.venues;
```

Comment:
There may be good reasons for it, but curious what the reason needs to be for having so many fine grained selectors here?

If we're going to do this, there should be a 'base selector' for `state.animatemap` that is then used by all of these other 'sub-selectors'